SECURITY.md

```diff
@@ -1,27 +1,77 @@
 ## Reporting a Vulnerability
 
-- If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/) or directly via email to aws-security@amazon.com. Please do **not** create a public GitHub issue.
-
-- For Security-CVE related fix - 
-    - For direct dependency - Use ```yarn upgrade package``` to update the package and in order to enforce as sub-deps please add nested-dep step2.
-
-    - For nested dependency/sub-deps - In order to enforce package above Vx.y.z, we can add version in the resolutions [section](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) for all the package sub-deps or specific package sub-dep. For more on version updates please see 
-[Why](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-why-would-you-want-to-do-this) and [How](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-how-to-use-it) to upgrade.
-    - To add the CVEs fix to previous versions, add label ex: backport 1.x. 
-  
-      ```
-      Example: foobar@1.x vulnerable package and 1.y is the fix 
-      step 1: 
-      For direct dependency checks: 
-              run: yarn upgrade foobar@1.y to update the package.json 
-              and yarn install to update the yarn.lock file
-      Step 2.
-      Check for sub deps foobar in other package. 
-              If foobar@1.x exists for subdeps in yarn.lock file
-              Then edit the package.json file and add **/foobar@1.y in resolution section as shown below to enforce the 1.y.
-                'resolutions': { "**/foobar": "^1.y", 
-                                 "**/foo": "^2.x" ,
-                                 "**/bar": "^3.k"}
-              Then run: yarn install for updating yarn.lock file
-
-    
\ No newline at end of file
+If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/) or directly via email to aws-security@amazon.com. Please do **not** create a public GitHub issue.
```

Comment:
```suggestion
If you discover a potential security issue in this project, we ask that you notify AWS/Amazon Security via the [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/) or directly, via email, to aws-security@amazon.com. Please do **not** create a public GitHub issue.
```

Reply:
I think this paragraph is using a standard template for all repos. For example, here is the SECURITY.md from OpenSearch: https://github.com/opensearch-project/OpenSearch/blob/main/SECURITY.md

Reply:
I am not sure where they got that but the grammar errors are bad.

Reply:
Except for the missing comma after the first subordinate clause, the other suggestions are less black and white... and I wouldn't classify any of them as "bad grammar", but I'm not a native speaker so what do I know :)

@AMoo-Miki: If you feel strongly about the way the paragraph is stated, I'd recommend opening up a PR against the one @ananzh links above, so that we can all benefit from this feedback.